app/controllers/response_plans_controller.rb

```diff
@@ -6,39 +6,32 @@ class ResponsePlansController < ApplicationController
   before_action :authenticate_officer!
   before_action :authorize_admin, except: [:index, :show]
 
-  def new
-    @response_plan = ResponsePlan.new(person: Person.new)
-  end
-
   def create
-    id = response_plan_params[:person_attributes][:id]
-    person = id ? Person.find(id) : nil
-
-    @response_plan = ResponsePlan.new(person: person, author: current_officer)
-    @response_plan.assign_attributes(response_plan_params)
+    person = Person.find(params.fetch(:person_id))
 
-    if @response_plan.valid?
-      @response_plan.person.save && @response_plan.save
-      redirect_to(
-        person_path(@response_plan.person),
-        notice: t("response_plans.create.success", name: @response_plan.person.name),
-      )
+    if person.active_response_plan
+      plan = person.active_response_plan.deep_clone(
+        except: [
+          :approver_id,
+          :approved_at,
+          :submitted_for_approval_at,
```

Comment:
Need to remove these, because otherwise it would have the same submitted/approved status as the original response plan.